graphics/blit-atari.cpp

```diff
@@ -40,24 +41,31 @@ void Surface::create(int16 width, int16 height, const PixelFormat &f) {
 	w = width;
 	h = height;
 	format = f;
-	pitch = w * format.bytesPerPixel;
+	pitch = (w * format.bytesPerPixel + 15) & 0xfff0;
 
 	if (width && height) {
-		if (VgetMonitor() == MON_VGA && Getcookie(C_SupV, NULL) == C_FOUND)
-			pixels = (void*)ct60_vmalloc(width * height * format.bytesPerPixel);
-		else
-			pixels = calloc(width * height, format.bytesPerPixel);
-		assert(pixels);
+		if (VgetMonitor() == MON_VGA && Getcookie(C_SupV, NULL) == C_FOUND) {
+			pixelsUnaligned = (void*)ct60_vmalloc(height * pitch + 15);
+			assert((uintptr)pixelsUnaligned >= 0xA0000000);
+		} else {
+			pixelsUnaligned = ::malloc(height * pitch + 15);
```

Comment:
If it's available, it may be simpler to make use of `aligned_alloc`.

Reply:
To my surprise, it is. Thanks for suggestion.